java-checks/src/test/files/checks/security/CipherBlockChainingCheck.java

```diff
@@ -0,0 +1,89 @@
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import javax.crypto.BadPaddingException;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.IvParameterSpec;
+
+class A {
+
+  SecureRandom random = new SecureRandom();
+  private byte[] bytes2 = "111".getBytes();
+  {
+    random.nextBytes(bytes2);
+  }
+  IvParameterSpec iv = new IvParameterSpec(bytes2); // Compliant FP, will be fixed with SE engine
+
+  void foo1()
+    throws InvalidKeyException, InvalidAlgorithmParameterException, NoSuchAlgorithmException, NoSuchPaddingException, UnsupportedEncodingException, IllegalBlockSizeException,
```

Comment:
What does this list of exceptions bring to the test? Can't we replace it with `throws Exception`?